encfs/FileUtils.cpp

```diff
@@ -1740,12 +1740,15 @@ void unmountFS(const char *mountPoint) {
   unmount(mountPoint, MNT_FORCE);
 #endif
 #ifdef __CYGWIN__
-  if(fork() == 0)
-  {
-    execl("/usr/bin/pkill", "/usr/bin/pkill", "-f", string("(^|/)encfs .*/.* ").append(mountPoint).append("?( |$)").c_str(), (char *)0);
-  }
+  pid_t pid;
   int status;
-  wait(&status);
+  if ((pid = fork()) == 0) {
+    execl("/usr/bin/pkill", "/usr/bin/pkill", "-INT", "-f", string("(^|/)encfs .*/.* ").append(mountPoint).append("?( |$)").c_str(), (char *)0);
+    _Exit(127);
```

Comment:
Code after execl is never reached!

Reply:
Yep but this is to make the child to exit if `execl` fails 👍

Reply:
Good point. But shouldn't we print an error message in that case?

Reply:
Yep, improved with log messages !